core/tests/runner/mod.rs

```diff
@@ -0,0 +1,94 @@
+use unimarkup_commons::test_runner::{
+    self, as_snapshot::AsSnapshot, snap_test_runner::SnapTestRunner, test_file,
+};
+use unimarkup_core::Unimarkup;
+
+use crate::snapshot::Snapshot;
+
+pub fn test_core_snapshots() -> Vec<libtest_mimic::Trial> {
+    let tests_path = unimarkup_commons::crate_tests_path!();
+
+    let test_cases = test_runner::collect_tests(
+        tests_path.join("spec/markup"),
+        tests_path.join("spec/snapshots/"),
+        "markup",
+    );
+
+    let mut test_runs = Vec::with_capacity(test_cases.len());
+
+    for case in test_cases {
+        let spec_test_name = format!("{}::spec::{}", module_path!(), case.test.name.as_str());
+        let snap_test_name = format!("{}::snap::{}", module_path!(), case.test.name.as_str());
+
+        let case2 = case.clone();
```

Comment:
```suggestion
        let cloned_case = case.clone();
```

Because `case2` sounds like it would be a different case.

Reply:
You're right, I renamed it now!